ndcube/mixins/plotting.py

```diff
@@ -82,10 +86,13 @@ def plot(self, axes=None, plot_axes=None, axes_coordinates=None,
             elif naxis == 2 and 'y' in plot_axes:
                 ax = self._plot_2D_cube(plot_wcs, axes, plot_axes, axes_coordinates,
                                         axes_units, data_unit, **kwargs)
-            else:
+            elif sunpy_available:
```

Comment:
I think it would be cleaner to shuffle this check into the `_animate_cube` method as then someone could overload it to animate a cube not with sunpy.

In fact going one step further than that, you could just put the import into that method.

Reply:
Although having imports in the middle of the code is typically frowned upon, in this case I think that's a great idea!

Reply:
Agreed, it's the kind of thing that I don't generally like to do, but it's definitely cleaner here. I kept the bit of added context for the ImportError